packages/codemod/src/utils/prop-utils.ts

```diff
@@ -1,31 +1,129 @@
-import { ASTPath, JSCodeshift, JSXIdentifier, JSXOpeningElement } from "jscodeshift";
+import {
+  ASTPath,
+  Collection,
+  JSCodeshift,
+  JSXAttribute,
+  JSXElement,
+  JSXIdentifier,
+  JSXOpeningElement,
+  Literal
+} from "jscodeshift";
+import { logPropMigrationError } from "./report-utils";
 
 /**
  * Updates a prop name in a JSX element
  */
 export function updatePropName(
-  elementPath: ASTPath<JSXOpeningElement>,
+  j: JSCodeshift,
+  elementPath: ASTPath<JSXElement>,
   propsNamesMappingOldToNew: Record<string, string>
 ): void {
-  const attributes = elementPath.node?.attributes;
-  if (!attributes) return;
-
-  attributes.forEach(attr => {
-    if (
-      attr.type === "JSXAttribute" &&
-      attr.name.type === "JSXIdentifier" &&
-      propsNamesMappingOldToNew[attr.name.name]
-    ) {
-      attr.name.name = propsNamesMappingOldToNew[attr.name.name];
-    }
-  });
+  j(elementPath)
+    .find(JSXOpeningElement)
+    .find(JSXIdentifier)
+    .forEach(identifierPath => {
+      if (propsNamesMappingOldToNew[identifierPath.node.name]) {
+        identifierPath.node.name = propsNamesMappingOldToNew[identifierPath.node.name];
+      }
+    });
 }
 
 /**
  * Checks for whether a prop is used in a JSX element
  */
-export function isPropExists(j: JSCodeshift, elementPath: ASTPath<JSXOpeningElement>, propName: string): boolean {
-  const attributes = elementPath.node?.attributes;
-  if (!attributes) return false;
```

Comment:
a mistake 😶

Reply:
the `find` of jscodeshift is taking care of it instead of the condition, it would return an empty collection instead of manipulating the code directly